### ISSUE TITLE
v2.5.0

### ISSUE THREAD

**Issue by watery** [bitnine-oss/agensgraph-docker#10]
Hello,
is the v2.5.0 Docker image I found on the Docker Hub ready to be used? I don't see any 2.5.0 among this repository _releases_.

However, I ran it with the following command:

```
docker run -d \
    --name agensgraph-bitnine-2.5.0 \
    --volume agens-2.5.0-data:/var/lib/postgresql/data \
    --restart unless-stopped \
    --publish 15432:5432 \
    -e POSTGRES_PASSWORD=mypassword \
    -e PGDATA=/var/lib/postgresql/data/pgdata \
    bitnine/agensgraph:v2.5.0
```

It seems to start fine, but the logs are full of:

```
2021-12-07 17:23:04.500 UTC [54] ERROR:  invalid value for parameter "graph_path": "agens"
2021-12-07 17:23:04.500 UTC [54] DETAIL:  graph "agens" does not exist.
2021-12-07 17:23:04.500 UTC [54] STATEMENT:   SET GRAPH_PATH=agens
```

Here's the full log since starting the container, it looks fine to me (but I mo PostgreSQL expert):

```
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting default timezone ... UTC
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
sh: locale: not found
2021-12-07 17:23:01.313 UTC [30] WARNING:  no usable system locales were found
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2021-12-07 17:23:02.174 UTC [35] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2021-12-07 17:23:02.222 UTC [36] LOG:  database system was shut down at 2021-12-07 17:23:01 UTC
2021-12-07 17:23:02.233 UTC [35] LOG:  database system is ready to accept connections
 done
server started

/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down...2021-12-07 17:23:02.271 UTC [35] LOG:  received fast shutdown request
.2021-12-07 17:23:02.275 UTC [35] LOG:  aborting any active transactions
2021-12-07 17:23:02.277 UTC [35] LOG:  background worker "logical replication launcher" (PID 42) exited with exit code 1
2021-12-07 17:23:02.277 UTC [37] LOG:  shutting down
2021-12-07 17:23:02.341 UTC [35] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2021-12-07 17:23:02.400 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2021-12-07 17:23:02.400 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2021-12-07 17:23:02.430 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2021-12-07 17:23:02.488 UTC [47] LOG:  database system was shut down at 2021-12-07 17:23:02 UTC
2021-12-07 17:23:02.512 UTC [1] LOG:  database system is ready to accept connections
2021-12-07 17:23:04.500 UTC [54] ERROR:  invalid value for parameter "graph_path": "agens"
2021-12-07 17:23:04.500 UTC [54] DETAIL:  graph "agens" does not exist.
2021-12-07 17:23:04.500 UTC [54] STATEMENT:   SET GRAPH_PATH=agens
2021-12-07 17:23:14.513 UTC [55] ERROR:  invalid value for parameter "graph_path": "agens"
2021-12-07 17:23:14.513 UTC [55] DETAIL:  graph "agens" does not exist.
2021-12-07 17:23:14.513 UTC [55] STATEMENT:   SET GRAPH_PATH=agens
[...]
```

**Comment by yjy44** [comment]
I'm really sorry to late..
Before you set graph path, you create graph path.
ex) create graph agens;
This logs, I can't find create graph path. Did you create graph path?

**Comment by watery** [comment]
I am no more on the project that was using Agensgraph, so I can't do any further test.

However, this is a Docker image that I pulled from the public hub: I would expect it to bring up and set up a full working containerized environment ready to be used.
But judging by my first post, I guess I just ran that `docker run` command without any extra operation, hence it looks like it's the image itself that's running the `SET GRAPH_PATH` directive.
